Jenkinsfile.promote

```diff
@@ -76,6 +96,83 @@ pipeline {
                 }
             }
         }
+        stage('Merge PR') {
+            when {
+                expression { return isRelease() && getDeployProperty('kogito-cloud-operator.pr.link') != ''}
+            }
+            steps{
+                script {
+                    githubscm.mergePR(getDeployProperty('kogito-cloud-operator.pr.link'))
+                    githubscm.pushObject("origin", getBuildBranch())
+                }
+            }
+        }
+        stage('Git tag') {
+            when {
+                expression {
+                    return getGitTag() != ""
+                }
+            }
+            steps {
+                script {
+                    sh "git pull origin ${getBuildBranch()}"
+                    githubscm.tagRepository(getGitTag(), env.BUILD_TAG)
+                    githubscm.pushObject("origin", "--tags ${getGitTag()}")
+                }
+            }
+        }
+        // stage("Create PR on operatorhub") {
+        //     when {
+        //         expression { return isRelease() }
+        //     }
+        //     steps {
+        //         script {
+        //             // This stage will be done in separate issue: 
+        //             // Any problem occuring on those PRs will be backported to operator if needed, for the next version
+
+        //             // // Openshift OperatorHub
+        //             // dir("operatorhub"){
+        //             //     // Example https://github.com/operator-framework/community-operators/pull/1876
+
+        //             //     checkout(githubscm.resolveRepository("community-operators", "operator-framework", master, false))
+        //             //     githubscm.forkRepo(getPRAccountCredsId())
+        //             //     githubscm.createBranch(getProjectVersion())
+        //             //     // TODO Copy csv files from operator
+        //             //     githubscm.commitChanges(getCommitMessage(), ".")
+        //             //     githubscm.pushObject("origin", getProjectVersion(), getPRAccountCredsId())
+        //             //     pr = githubscm.createPR(getCommitMessage(), "Generated by build ${BUILD_TAG}: ${BUILD_URL}", "master", getPRAccountCredsId())
+        //             //     echo "Created PR ${pr}"
+        //             // }
+
+        //             // // Operator io kubernetes
+        //             // dir("kubernetes"){
+        //             //     echo "TODO waiting for example PR from zanini"
+        //             //     //checkout(githubscm.resolveRepository("community-operators", , getBuildBranch(), false))
+        //             // }
+        //         }
+        //     }
+        // }
+        stage('Set next version') {
+            when {
+                expression { return isRelease() }
+            }
+            steps {
+                script {
+                    nextVersion = getNextVersion()
```

Comment:
`getNextVersion()` doesn't currently exist?

Reply:
oups you are right, gonna create it ...